test/controllers/AmendConfirmationControllerSpec.scala

```diff
@@ -44,8 +45,10 @@ class AmendConfirmationControllerSpec extends SpecBase {
 
         status(result) mustEqual OK
 
+        val checkYourAnswersHelper = new CheckYourAnswersHelper(answers)
+
         contentAsString(result) mustEqual
-          view(claimId)(request, messages).toString
+          view(claimId, checkYourAnswersHelper.getAmendCheckYourAnswerSections)(request, messages).toString
```

Comment:
RED: I was expecting to see a set of tests here for "AmendConfirmation Controller for summary view" in the same way that ConfirmationControllerSpec has

Reply:
Tests for summary view added